greenplumpython/expr.py

```diff
@@ -64,6 +76,23 @@ def __str__(self) -> str:
         return str(self.left) + " " + self.operator + " " + str(self.right)
 
 
+class UnaryExpr(Expr):
+    def __init__(self, operator: str, right: Expr, as_name: Optional[str] = None):
+        super().__init__(as_name=as_name)
+        self.operator = operator
+        self.right = right
+
+    def __str__(self) -> str:
```

Comment:
Why do you add `As "xxxx"` in this function? I think the `as_name` should be handled by the parent class `Expr`?

Reply:
Changed it to use `as_name`